app/views/shared/_rollbar.html

```diff
@@ -0,0 +1,54 @@
+<script type="text/javascript">
+// Rollbar reports errors in production.
+// Always load it, and then as errors come in decide whether to report.
+// window.ENV_FOR_JS, which should be on the page on boot.
+
+// If the application code didn't load for some reason, report errors.
+// Only silence reporting if it loaded and there's a key that explicitly
+// disables reporting (eg., for development mode).
+// Returns true to ignore the error.
+function checkIgnore(isUncaught, args, payload) {
+  if (window.ENV_FOR_JS.shouldReportErrors === false) return true;
+  return false;
+}
+
+// Add in Rails env info to payload if we're reporting and it exists.
+function transform(payload) {
+  payload.environment = window.ENV_FOR_JS.railsEnvironment;
+  payload.deploymentKey = window.ENV_FOR_JS.deploymentKey;
+  payload.districtKey = window.ENV_FOR_JS.districtKey;
+}
+
+// Sets global configuration for the Rollbar error reporting library,
+// based on the environment, and then loads the Rollbar code or not.
+var _rollbarConfig = { // eslint-disable-line no-var
+  accessToken: window.ENV_FOR_JS.rollbarJsAccessToken,
```

Comment:
The only substantive change here is to read the token from `window.ENV_FOR_JS`, and to read from that in other places instead of `window.shared.Env`.  This code doesn't use `readEnv()` since this isn't normal JS code - it's not compiled by Webpack and is inlined into the HTML since it's on the critical path for loading and error reporting.